kolibri/plugins/document_pdf_render/assets/src/views/index.vue

```diff
@@ -39,6 +62,9 @@
       mimicFullscreen() {
         return !this.fullscreenAllowed && this.isFullscreen;
       },
+      targetTime() {
+        return this.totalPages * 30;
```

Comment:
Now that we're able, would it make sense to combine view time with amount scrolled? e.g. require scrolling all the way, but also having a min time?